### DISCUSSION TITLE
a-cheap-no-frills-opnsense-box/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#224] [Announcements]
# a-cheap-no-frills-opnsense-box/

If you're looking to build a small, quiet and cheap box to run OPNsense on then I have a build you might be interested in. This box has been happily running as my firewall since July 2019. I originally documented the build in this forum post over at serverbuilds.net

https://blog.ktz.me/a-cheap-no-frills-opnsense-box/

**Comment by valentin-voica** [comment]
I've been using OPNsense since 2020. As I have a small, 6U furniture cabinet, I needed a 1U rack mount chassis: Supermicro SC502L-200B; I replaced the fan inside the PSU with a Noctua NF-A4x20 FLX. The motherboard is Supermicro X7SPE-HF-D525 with a soldered Intel Atom D525 CPU, 4GB DDR3, and dual 1G NIC. The HDD is 80GB Barracuda 7200.10 (under a fiver of eBay). The whole affair was about £120 (eBay).

I love it, The only plugins I'm using is Wireguard so 4GB are more than enough for my needs. Also the 1GB NICs is not a bottle-neck for the five VLANs traversing them.

**Comment by xcojonny** [comment]
nice :) How high is the power consumption on average?

**Reply by valentin-voica** [reply]
I'm afraid I don't know the consumption. You made me curious and tried to find out by plugging the firewall into a Local Bytes smart plug which I briefly used a while ago. After reconnecting it to the WLAN and updating the firmware it failed in delivering power - tried re-imaging with ESPhome and now I can't even access it's web GUI and HA can't update the f/w from "minimal" to ESPhome full..

I'll let you know if this changes.

**Reply by valentin-voica** [reply]
I have finally managed to check the power consumption for my OPNsense box; it averages at 34W. Same value measured using a LocalBytes Power Monitoring Plug (UK) ESP, and a Shelly Plus 1PM Mini.

This is actually the same as my home server (ASRock Rack E3C224D2I motherboard + Intel Xeon E3-1240 v3 CPU + 16GB of RAM, + 1x 120GB SSD + 1x 1TB HDD + 1x 500GB HDD all housed by a nice In Win  IW-RF100S case) running a dozen containers in Swarm mode.

It's high time to ditch the "Supermicro"  server and get a mini-PC for OPNsense...
